config-builder/src/test/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/PolicyEntityBuilderTest.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.tasks.zip.builder;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PolicyEntityBuilderTest {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add some tests to this class. ([squid:S2187](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2187))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=98&issues=AWajX2esrChX7_AlPFEW&open=AWajX2esrChX7_AlPFEW)

[ISSUE_KEY]: AWajX2esrChX7_AlPFEW